### PR TITLE
feat: make cli commands consumable in Ory Cloud

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

**Comment by aeneasr** [comment]
@zepatrik fyi

File: docs/sidebar.json [hunk 1]
```diff
@@ -50,6 +50,31 @@
         "cli/kratos-version"
       ]
     }, 
+    {
```
Comment (zepatrik) [inline docs/sidebar.json:53]: why is this here two times now?

File: go.mod [hunk 2]
```diff
@@ -7,12 +7,14 @@ replace github.com/markbates/pkger => github.com/falafeljan/pkger v0.17.1-0.2020
 
 replace gopkg.in/DataDog/dd-trace-go.v1 => gopkg.in/DataDog/dd-trace-go.v1 v1.27.1-0.20201005154917-54b73b3e126a
 
+// Use the internal httpclient which can be generated in this codebase but mark it as the
```
Comment (zepatrik) [inline go.mod:10]: Big brain

File: docs/sidebar.json (hunk 1 above)
Comment (aeneasr) [reply docs/sidebar.json:53]: probably another failure in the automation

**Comment by zepatrik** [comment]
When I run `make sdk` now it deletes the `go.mod` but does not create a new one. Also it was a pain to upgrade swagger, I first had to remove the rewrite of the client and then change the version number to some release that exists. Can you maybe have a look on that @aeneasr ? Next week is fine though, I thought I'd debug the sidebar cli docs generation issue, but won't do that this week now.

**Comment by aeneasr** [comment]
Yeah I'll take a look!
